shared/fs/header/add-new.js

```diff
@@ -74,7 +74,7 @@ const propsToMenuItems = (props: AddNewProps) => {
       icon: 'iconfont-upload',
     })
   props.pickAndUploadPhoto && props.pickAndUploadVideo && items.push('Divider')
-  items.push({title: 'New folder', onClick: props.newFolderRow, icon: 'iconfont-folder-new'})
+  items.push({title: 'Create New folder', onClick: props.newFolderRow, icon: 'iconfont-folder-new'})
```

Comment:
I think we may want to lowercase the "N" in "New" though so the text is `Create new folder`

Reply:
Cool, let me change that